src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+export const Input = ({ onEditComplete, defaultValue }) => {
```

Comment:
```suggestion
export const Input = ({ onEditComplete, defaultValue="" }) => {
```
とすることでdefaultValueに初期値を渡すことができます。
案件などに参加した際にtypescriptを使用するのですがその時に意識すべき型という概念があります。明示的に空を指定することでdefaultValueはstring型(文字列)だと認識されるのですが、今の状態のままだとstringまたはundefined型になるので、型の一貫性の意味でも個人的には指定しておきたいです。

Reply:
ありがとうございました！よくわかりました。